### PR TITLE
SCRIPTS: Add management features to wsmux

### PR DESCRIPTION
**Pull request by sentree**
This implements the functionality requested in uf-mil/SubjuGator#239. It also adds the functionality to bind to the current workspace if none is passed with the `-b` argument and the ability to unbind an SSH client with the `-u` argument.

### CODE REVIEW

File: scripts/wsmux.sh [hunk 1]
```diff
@@ -99,30 +105,97 @@ wsmux() {
...
+
+				if [[ "$SELECTION" != "false" ]]; then
+					for FILE in ~/$SELECTION/src/*; do
+
+						# Verify that the directory is a Git repository
+						if [[ -d $FILE/.git ]]; then
+							cd $FILE
+
+							# Verify that the repository's working directory is clean
+							if $(git diff-index --quiet HEAD --); then
```
Comment (kev-the-dev) [inline scripts/wsmux.sh:185]: So is this checking for the exit code? Looks like this command returns exit code 1 for an unclean working directory. So isn't this the opposite of what you want (assuming if runs for positive exit codes)

File: scripts/wsmux.sh [hunk 2]
```diff
@@ -99,30 +105,97 @@ wsmux() {
...
+
+								# Fetch the most recent changes from upstream
+								git fetch upstream
+
+								# Force the creation of a new master branch based on upstream
+								git checkout -B master upstream/master
```
Comment (kev-the-dev) [inline scripts/wsmux.sh]: Be sure to handle submodules. I think ```git submodule update``` will do it.

**Review by kev-the-dev — CHANGES_REQUESTED** [review]
I ain't no bash boi but I don't think this will have the desired behavior.

File: scripts/wsmux.sh (hunk 1 above)
Comment (sentree) [reply scripts/wsmux.sh:185]: In bash, an exit code of 0 will be taken as `true` for an if statement. So if the working directory is clean, the command will return 0 and the update will proceed.

File: scripts/wsmux.sh (hunk 2 above)
Comment (sentree) [reply scripts/wsmux.sh]: Are you sure that won't update them to the latest version of their origin repository?

Comment (sentree) [reply scripts/wsmux.sh]: This has been added and tested.

**Review by kev-the-dev — APPROVED** [review]

**Review by DSsoto — APPROVED** [review]

**Pull request merged**